Source/Git/git-info.env

```diff
@@ -539,13 +539,50 @@ _t4dSrcGitInfoRelativeRootPath(){
 }
 
 _t4dSrcGitInfoProjectKEY(){
+# Generated From $Tools4Dev_PATH/Engine/G4d/templates.env
+# v1.8.0
 ###### _t4dSrcGitInfoProjectKEY
-#   Return git project KEY
-#   Usage : _t4dSrcGitInfoProjectKEY
+#   - Name:
+#       _t4dSrcGitInfoProjectKEY
+###
+#   - Synopsis:
+#       return git project key
+###
+#   - Note:
+#       <Required>
+#       [Optionnal]
+#       {XXXXXXX}       = Value To Be Changed, if {URL} then replace {URL} with a valid URL
+###
+###### DOCUMENTATION BLOC #####
+########### CODE BLOC #########
+    local _t4dErrorCode=1
```

Comment:
Can remove unused errorcode

Reply:
nop, for now we don't have to manage errorCode but in the future, this can become important to identify errorCode so even if today it always return 1 it's better to have it readyToBeUse